scripts/jest/preprocessor.js

```diff
@@ -78,7 +75,7 @@ module.exports = {
       const isInDevToolsPackages = !!filePath.match(
         /\/packages\/react-devtools.*\//
       );
-      const testOnlyPlugins = [pathToBabelPluginAsyncToGenerator];
+      const testOnlyPlugins = [];
       const sourceOnlyPlugins = [];
```

Comment:
These can just be removed now as both sides of the condition are empty :)

Reply:
`sourceOnlyPlugins` is used on line 81. My rationalization for leaving `testOnlyPlugins` is so the next person knows where to add a test-only plugin; the real reason is because I don't want to show up in the git blame for the rest of this code.

Reply:
Oh, I missed that it get's mutated. Thanks!